testr/runner.py

```diff
@@ -45,7 +45,7 @@ def testr(*args, **kwargs):
         kwargs.setdefault(kwarg, True)
 
     # test() function looks up the calling stack to find the calling package name.
-    # It will be two levels up.
+    # It will be three levels up including the stdout_wrapper test_wrapper function.
     kwargs['stack_level'] = 2
```

Comment:
you changed the number in the comment but not in the code. Why change it in the comments if it has no effect? If the comment needs to be changed, it would be to explain a bit more what the number is, because clearly right now I do not remember.

Reply:
Good catch Javier.  I reverted back to the previous level in the code when I moved from decorating the full test function to just using a context manager for the pytest calls, but did not revert the comment correctly.